src/Components/Pet.js

```diff
@@ -1,55 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
 import '../Styles/Pet.scss';
+import { useQuery, gql } from '@apollo/client';
 import Header from './Header';
+import { useParams } from 'react-router-dom';
+import OwnerInformation from './OwnerInformation';
+import PetInformation from './PetInformation';
 
 
+const GET_SINGLE_PET = gql`
+query getPetById($id: ID!) {
+  getPetById(id: $id) 
+  {
+    id
+    name
+    age
+    gender
+    description
+    species
+    ownerStory
+    ownerEmail
+    ownerName
+    applications {
+      name
+      email
+      description
+    }
+  }
+}
+`
+
 const Pet = () => {
+  const [showOwnerInfo, setOwnerInfo] = useState(false);
+  
+  let { id } = useParams();
+  const { loading, error, data } = useQuery(GET_SINGLE_PET, {variables: {id}})
```

Comment:
Key: variables. Value: OBJECT. Took us a long time to realize this and that we needed { }. Once we did, magic. Our info came through.